pkg/container/qemu_runner.go

```diff
@@ -1221,28 +1221,90 @@ func getAvailableMemoryKB() int {
 
 		for s.Scan() {
 			var n int
-			if nItems, _ := fmt.Sscanf(s.Text(), "MemTotal: %d kB", &n); nItems == 1 {
+			// Try to get MemAvailable first (available on newer kernels)
+			if nItems, _ := fmt.Sscanf(s.Text(), "MemAvailable: %d kB", &n); nItems == 1 {
```

Comment:
This is giving some problems on Linux, can we revert this to the original and only modify the MacOS behaviour?

Reply:
CI problem is unrelated to this, disregard my comment